cloud/disk_manager/internal/pkg/clients/nbs/testcommon/common.go

```diff
@@ -0,0 +1,156 @@
+package testcommon
+
+import (
+	"bytes"
+	"context"
+	"hash/crc32"
+	"math/rand"
+	"time"
+
+	"github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/clients/nbs"
+	"github.com/ydb-platform/nbs/cloud/disk_manager/internal/pkg/types"
+	"github.com/ydb-platform/nbs/cloud/tasks/logging"
+	"golang.org/x/sync/errgroup"
+)
+
+////////////////////////////////////////////////////////////////////////////////
+
+func NewContext() context.Context {
+	return logging.SetLogger(
+		context.Background(),
+		logging.NewStderrLogger(logging.DebugLevel),
+	)
+}
+
+////////////////////////////////////////////////////////////////////////////////
+
+func FillDisk(
+	nbsClient nbs.Client,
+	diskID string,
+	diskSize uint64,
+) (uint32, uint64, error) {
+
+	return FillEncryptedDisk(nbsClient, diskID, diskSize, nil)
+}
+
+func FillEncryptedDisk(
```

Comment:
Add ctx to the signature and move whole method to clients/nbs/testing_client.go, please